src/optionsMenu/helpers/VUMeter.tsx

```diff
@@ -86,19 +102,31 @@ export function VUMeter(props: VUMeterProps) {
       ctx.fillStyle = "#e8d120";
     }
 
-    const valueOffset =
-      (Math.max(peak, props.range[0]) - props.range[0]) / valueRange;
+    const valueOffsetL =
+      (Math.max(peakL, props.range[0]) - props.range[0]) / valueRange;
 
-    ctx.fillRect(0, 0, valueOffset * width, height - 10);
+    let valueOffsetR = 0;
+    if (props.stereo) {
+      valueOffsetR =
+        (Math.max(peakR, props.range[0]) - props.range[0]) / valueRange;
+      ctx.fillRect(0, 0, valueOffsetL * width, height / 2 - 7);
+      ctx.fillRect(0, height / 2 - 6, valueOffsetR * width, height / 2 - 7);
+    } else {
+      valueOffsetR = valueOffsetL;
+      ctx.fillRect(0, 0, valueOffsetL * width, height - 14);
+    }
 
     ctx.fillStyle = "#fff";
     for (let i = 0; i < 10; i++) {
       const value = (props.range[0] + valueRange * (i / 10)).toFixed(0);
-      ctx.fillText(value, width * (i / 10), height - 7);
+      ctx.fillText(value, width * (i / 10), height - 2);
     }
-  }, [peak, props.range, props.greenRange]);
+  }, [peakL, peakR, props.range, props.greenRange, props.stereo]);
 
   const { value, range, greenRange, source, ...rest } = props;
 
+  // Can't put boolean values into HTML DOM.
+  delete rest.stereo;
```

Comment:
```suggestion
  const { value, range, greenRange, source, stereo, ...rest } = props;
```
Consistency (although does this make the linter screm?)

Reply:
Doh! Such a simple solution.